.github/workflows/push-master-ci.yml

```diff
@@ -13,6 +13,36 @@ env:
   ext-dir: $GITHUB_WORKSPACE/external/install/
 
 jobs:
+  upgrade-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Read VERSION
+        id: read
+        uses: juliangruber/read-file-action@v1
+        with:
+          path: ./VERSION
+      - name: trim VERSION
+        id: set
+        run: |
+          echo "::set-output name=version::${{ steps.read.outputs.content }}"
+          [[ "${{ github.ref_name }}" == "master" ]] && echo "::set-output name=fragment::feature" || echo "::set-output name=fragment::bug"
```

Comment:
Can you add a comment explaining how to set the `bug` and `feature` keys ? Should it be in the commit message, or in the PR label ?